backend/canvas_app_explorer/views.py

```diff
@@ -36,7 +38,8 @@ def list(self, request: Request) -> Response:
 
         logger.debug('available_tools: ' + ', '.join([tool.__str__() for tool in available_tools]))
         available_tool_ids = [t.id for t in available_tools]
-        queryset = models.LtiTool.objects.filter(canvas_id__isnull=False, canvas_id__in=available_tool_ids)\
+        queryset = models.LtiTool.objects.filter(Q(canvas_id__isnull=False, canvas_id__in=available_tool_ids)#).order_by('name')
```

Comment:
Should remove this unused comment. 

Nitpick: The new `.order_by` could probably also be better aligned or put on the preceding line if it fits.

Reply:
Done!